mealie/lang/messages/en-US.json

```diff
@@ -4,6 +4,7 @@
     },
     "recipe": {
         "unique-name-error": "Recipe names must be unique",
+        "recipe-created": "Recipe created",
```

Comment:
Should this be "Recipe Created"? (vs "Recipe created")

Reply:
Fixed it